inbox/models/message.py

```diff
@@ -306,6 +306,27 @@ def create_from_synced(cls, account, mid, folder_name, received_date,
                     setattr(msg, field, [])
                     msg._mark_error()
 
+        self.subject = self.subject if self.subject else ''
+        self.snippet = self.snippet if self.snippet else ''
+        self.body = self.body if self.body else ''
+
+        batch_input = [
+            {"plaintext": self.subject},
+            {"plaintext": self.snippet},
+            {"plaintext": self.body},
+            {"plaintext": body_string},
```

Comment:
what is body_string? don't see it defined anywhere and don't see it used after

Reply:
is's raw data of email. It's defined above. It's parsed and we get subject, snippet, body and other staff.